Initial commit for soc_004a script (#119)
resource-watch/data-pre-processing: Pending_updates/soc_004a_human_development_index/README.md

## Human Development Index Dataset Pre-processing
This file describes the data pre-processing that was done to [the Human Development Index (HDI)](http://hdr.undp.org/en/2016-report) for [display on Resource Watch](https://resourcewatch.org/data/explore/bea122ce-1e4b-465d-8b7b-fa11aadd20f7).

The source provided the data in a .xlsx format.

Below, we describe the steps used to reformat the table so that it is formatted correctly to upload to Carto.

> 1. Read in the data as a pandas dataframe and remove empty columns and rows.

Reviewer: ```suggestion
1. Read in the data as a pandas dataframe.
```

> 1. Read in the data as a pandas data frame and remove columns and rows with missing data.

Reviewer: We don't remove rows/columns with any missing data, but only when it is ALL missing
```suggestion
1. Read in the data as a pandas dataframe and remove empty columns and rows.
```

> 2. Give columns the correct headers.

Reviewer: ```suggestion
2. Remove header rows containing metadata, and give columns the correct headers.
```

> 3. Replace ".." with None.

Reviewer: I don't see this in the script - did I miss it?
```suggestion
```

Author: I actually ran the code...just added it to the script.

> 2. Remove header rows containing metadata, and give columns the correct headers.

Reviewer: ```suggestion
2. Remove rows containing metadata (headers, sub-sections names, footnotes, etc.).
3. Name columns based on the year, title, and unit provided in the metadata.
```
